open-apparel-registry/open-apparel-registry
# Rename OAR to OS Hub

## Overview

Updates any references to OAR or Open Apparel Registry _visible to a user or API consumer_ to OS Hub and Open Supply Hub respectively. This _includes_ renaming OAR ID _except_ in API responses, where it continues to be `oar_id` for now.

Connects #1902 

## Demo

![localhost_6543_ (3)](https://user-images.githubusercontent.com/4432106/175665678-eadcec91-4b68-47b1-afbb-949134d86e55.png)

## Notes

 - After discussion with @jwalgran, we decided to _not_ rename OAR -> OS Hub in any code or code comments for now, and to defer that until after the OS Hub beta period is over. The intent of this is to minimize the number of unnecessary differences between production and the beta site while we may have to target hotfixes to both branches. We will do a follow-on pull request after the beta period is over to remove all final references to OAR.
 - I made the commits renaming "OAR ID" to "OS Hub ID" separate since I wasn't sure which areas if any we might want to keep as OAR ID until we rename the API field itself

## Testing Instructions

* `scripts/server`
* Try to double-check to make sure I didn't miss any user-visible uses of OAR or Open Apparel Registry

## Checklist
